addon/components/google-recaptcha.js

```diff
@@ -0,0 +1,62 @@
+import Component from "@ember/component";
+import { isNone } from "@ember/utils";
+import { bind, later } from "@ember/runloop";
+import { next } from "@ember/runloop";
+import config from "ember-get-config";
+import getScript from "../utils/get-script";
+
+export default Component.extend({
+  /**
+   * Component based off ember-g-recaptcha ember component:
+   * https://github.com/algonauti/ember-g-recaptcha
+   */
+  classNames: ["g-recaptcha"],
+  sitekey: config.googleCaptchaKey,
+  renderReCaptcha() {
+    if (isNone(window.grecaptcha) || isNone(window.grecaptcha.render)) {
+      later(() => {
+        this.renderReCaptcha();
+      }, 500);
+    } else {
+      let container = this.element;
+      if (container) {
+        let widgetId = window.grecaptcha.render(container, {
+          sitekey: this.get("sitekey"),
+          callback: bind(this, "successCallback"),
+          "expired-callback": bind(this, "expiredCallback")
+        });
+        this.set("widgetId", widgetId);
+      }
+    }
+  },
+
+  resetReCaptcha() {
```

Comment:
it looks like this is only called in `expiredCallback`. why not bind this directly to the `expired-callback` property on the widget?

Reply:
Ah, yeah - this was leftover extensibility from the add-on, if you wanted to customize what happens when the reCaptcha expired, you could do it. But good point, we're not using it so I'll ditch it.